api/account/api.go

```diff
@@ -100,7 +100,7 @@ type Configuration struct {
 	AutoIdentificationVoucherEnabled      *bool           `json:"auto_identification_voucher_enabled"`
 	AutoIdentificationAccountingEnabled   *bool           `json:"auto_identification_accounting_enabled"`
 	AutoIdentificationPrepaymentRequired  *bool           `json:"auto_identification_prepayment_required"`
-	AllowedBackURL                        *string         `json:"allowed_back_url"`
+	AllowedBackURLs                       []string        `json:"allowed_back_urls"`
```

Comment:
Do we want to support only new version? Selfservice will be ready in the same time?

Reply:
Selfservice and Backoffice have been updated